src/lib/web3-utils.js

```diff
@@ -110,7 +113,7 @@ export function getNetworkName(chainId) {
 }
 
 export function isLocalOrUnknownNetwork(chainId) {
-  return getNetworkName(chainId) === DEFAULT_LOCAL_CHAIN
+  return getNetworkType(chainId) === DEFAULT_LOCAL_CHAIN
```

Comment:
let's also do the change [here](https://github.com/aragon/court-dashboard/blob/fa739200349b326cf4182daddc9bfb0ef10ec01e/src/components/Onboarding/OnboardingModal.js#L10)

Reply:
Done! https://github.com/aragon/court-dashboard/blob/activity/src/components/Onboarding/OnboardingModal.js#L10